build.sh

```diff
@@ -238,9 +258,9 @@ prepare_to_build_project "Dynamatic" "build"
 # CMake
 if should_run_cmake ; then
   cmake -G Ninja .. \
-      -DMLIR_DIR=polygeist/llvm-project/build/lib/cmake/mlir \
-      -DLLVM_DIR=polygeist/llvm-project/build/lib/cmake/llvm \
-      -DCLANG_DIR=polygeist/llvm-project/build/lib/cmake/clang \
+      -DMLIR_DIR=$POLYGEIST_DIR/llvm-project/build/lib/cmake/mlir \
```

Comment:
`$POLYGEIST_DIR` should be enclosed in quotes (`"`) otherwise any space in the Polygeist path will split arguments and cause the command to fail. The same logic applies to everywhere you use `$POLYGEIST_DIR` below.